GitHubTraining/Program.cs

```diff
@@ -30,6 +30,7 @@ static void Main(string[] args)
             cars.Add(new Car("Mazda", 2016, "Blue"));
             cars.Add(new Car("Nissan", 2010, "Red"));
             cars.Add(new Car("Bentley", 1950, "Red"));
+            cars.Add(new Car("Toyota", 2012, "White"));
```

Comment:
Looks good, but consider adding one more car.

Reply:
ok, I will add it.